pwd-service/src/main/java/org/pwd/domain/audit/Metric.java

```diff
@@ -6,9 +6,11 @@
  * @author bartosz.walacik
  */
 public enum Metric {
-    anyTitle (1, "Tytuł"),
-    alt      (2, "Text alternatywny"),
-    htmlLang (1, "Język strony")
+    alt      (1, "1.1.1 Text alternatywny"),
```

Comment:
Zakładałem, że 1.1.1 to element nazwy i niegdy nie będzie niezależnie
występował.
Jak rozdzielimy to będziemy musieli potem sklejać.
Jak uważasz, że lepiej to tak zróbmy.

2015-08-31 10:07 GMT+02:00 Bartosz Walacik <notifications@github.com>:

> In pwd-service/src/main/java/org/pwd/domain/audit/Metric.java
> <https://github.com/pwd-project/pwd/pull/50#discussion_r38291799>:
>
> > @@ -6,9 +6,11 @@
> >   * @author bartosz.walacik
> >   */
> >  public enum Metric {
> > -    anyTitle (1, "Tytuł"),
> > -    alt      (2, "Text alternatywny"),
> > -    htmlLang (1, "Język strony")
> > +    alt      (1, "1.1.1 Text alternatywny"),
>
> nie sklejamy ze sobą pól, ten identyfikator 1.1.1 musi trafić do
> odzielnego pola
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pwd-project/pwd/pull/50/files#r38291799>.
>